apps/src/templates/sectionProgress/progressTables/ProgressTableContentView.jsx

```diff
@@ -103,9 +103,11 @@ export default class ProgressTableContentView extends React.Component {
 
   columnWidthStyle(index) {
```

Comment:
Might be worth adding a comment somewhere around here describing when we expect columnWidths to be specified and when not and the overall behavior associated with each case.

Reply:
👍